inblay/report/purchase_order_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_purchaseorder_document_inherit" inherit_id="purchase.report_purchaseorder_document">
+        <xpath expr="//div[@id='informations']/div[last()]" position="after">
+            <div class="col-3 bm-2">
+                <strong>Currency:</strong>
+                <p t-field="o.currency_id" class="m-0"/>
+            </div>
+        </xpath>
+        <xpath expr="//div[@id='total']//table/tr[1]/td[1]" position="replace">
```

Comment:
- Isn't there another way to retrieve that element?
- Remember to, if replacing elements, to add a priority greater than 100

Reply:
Done.